tests/access-code-unique.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { test } from '@playwright/test';
+import { LoginPage  as UILoginPage } from '../src/pages/ui';
+import { SettingsPage } from '../src/pages/ui';
+const accessCode: string = 'e2e-testing' + Math.random();
```

Comment:
I think it would be best to create this variable in the method, it should be specific to every test run